runtime/queries/rust/highlights.scm

```diff
@@ -40,7 +40,7 @@
 ; ---
 
 (self) @variable.builtin
-(enum_variant (identifier) @type.enum.variant)
+(enum_variant (identifier) @constructor)
```

Comment:
Except for this change, I am fine with the rest of the PR as long as tracked to be at least partially reverted once semantic highlighting is implemented.